idrcloudclient.js

```diff
@@ -14,31 +14,44 @@
  * limitations under the License.
  */
 
-var request = require('request');
+var http = require('http');
+var https = require('https');
+var FormData = require('form-data')
```

Comment:
Minor adjustment for code quality, otherwise, change looks good
```suggestion
var FormData = require('form-data');
```